thrift/sqlblobs.thrift

```diff
@@ -276,3 +276,15 @@ struct ReplicationTaskInfo {
   34: optional binary newRunBranchToken
   38: optional i64 (js.type = "Long") creationTime
 }
+
+enum AsyncRequestType {
+  StartWorkflowExecutionAsyncRequest
+}
+
+struct AsyncRequestMessage {
+  10: optional string partitionKey
+  12: optional AsyncRequestType type
+  14: optional shared.Header header
+  16: optional string encoding
+  18: optional binary payload
+}
```

Comment:
what is the value of defining storage layer blobs in this idls repo? clients don't need to know about this.

Reply:
the kafka producer needs to encode the payload, and currently it uses thrift encoder in the codebase
https://github.com/uber/cadence/blob/master/common/messaging/kafka/producer_impl.go#L83

Reply:
Discussed offline. Basically cadence-idl repo is a place for shared proto/thrift files which both server and client sdk needs. Storage layer schemas don't need to be here because client doesn't know about those but since they are here for historical reasons we keep it that way for consistency.